.gitignore

```diff
@@ -25,3 +25,7 @@ packages/*/yarn.lock
 yarn-error.log
 !packages/utils/tests/fixtures/packages/node_modules
 i18n.import.ts
+packages/formatter-html/src/assets/js/scan/get-message.js
+packages/formatter-html/src/assets/js/scan/_locales
+packages/hint-axe/src/meta
```

Comment:
I think the approach so far has been for each package to have its own `.gitignore`. That should also make it easier to maintain and scope.